docs/architecture.md

```
@@ -1,73 +1,36 @@
 ## ディレクトリ構造
+- `bloom/api` : WebSocket シグナリング用メッセージ型（ClientToServer / ServerToClient、RelaySdp/Ice、ErrorCode）。
+- `bloom/core` : ルーム/参加者管理（最大 8 名）、UUID ベースの RoomId/ParticipantId、Join/Leave ロジック。
+- `bloom/ws` : `/ws` エンドポイントを持つ実サーバ。rate limit 1 秒 20 メッセージ/セッション、tracing で room_id/participant_id を付与。バイナリ `main.rs` が subscriber を初期化。
+- `syncer` : P2P 同期ライブラリ。SyncMessage Envelope v1（kind=pose/chat/voice/control.* / signaling.*）、BasicSyncer（1 リクエスト→複数イベント）、RateLimiter（1 秒 20 件/セッション）、WebRTC Transport（DataChannel label `sutera-data`、Pose unordered/unreliable、音声トラック/Opus 対応）。
+- `docs/specs` : 仕様群（20251125-bloom-signaling, 20251203-syncer-minimal-p2p ほか）。
 
-// TODO
+## SuteraVRを構成する要素（2025-12-18 現在）
 
-## SuteraVRを構成する要素
+### Bloom（シグナリング）
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)